.github/workflows/ci-build.yml

```diff
@@ -16,6 +16,7 @@ jobs:
           - { os: ubuntu-22.04, compiler: clang-15 }
           - { os: ubuntu-24.04, compiler: gcc-14   }
           - { os: ubuntu-24.04, compiler: clang    } # Clang 18
+          - { os: ubuntu-24.04-arm, compiler: clang   }
```

Comment:
I might prefer to see GCC here since it's still the default on Ubuntu/ARM.  Having both configurations would also be fine